tests/unit/array/storage/elastic/test_get.py

```diff
@@ -0,0 +1,54 @@
+from docarray import Document, DocumentArray
+import numpy as np
+import pytest
+
+
+def test_success_get_bulk_data(start_storage):
+    nrof_docs = 20000
```

Comment:
```suggestion
@pytest.mark.parametrize('nrof_docs', [10, 100, 10_000, 10_100, 20_000, 20_100])
def test_success_get_bulk_data(start_storage, nrof_docs):

```

let's just test different values.